packages/local-build-plugin/src/eject.ts

```diff
@@ -21,19 +21,36 @@ export class LocalExpoCliEjectProvider implements EjectProvider<Job> {
 
     const expoCliBinPath =
       process.env.EXPO_CLI_PATH ?? path.resolve(path.dirname(expoCliPackage), '../bin/expo.js');
-    logger.debug(`${expoCliBinPath} prebuild --non-interactive --platform ${job.platform}`);
-    await spawnAsync(
-      expoCliBinPath,
-      ['prebuild', '--non-interactive', '--platform', job.platform],
-      {
-        ...spawnOptions,
-        cwd: ctx.reactNativeProjectDirectory,
-      }
-    );
+    const prebuildCommand = this.getPrebuildCommand(job);
+    logger.debug(`${expoCliBinPath} ${prebuildCommand}`);
+    await spawnAsync('bash', ['-c', `${expoCliBinPath} ${prebuildCommand}`], {
+      ...spawnOptions,
+      cwd: ctx.reactNativeProjectDirectory,
+    });
 
     await spawnAsync(ctx.packageManager, ['install'], {
       ...spawnOptions,
       cwd: ctx.reactNativeProjectDirectory,
     });
   }
+
+  private getPrebuildCommand(job: Job): string {
+    let prebuildCommand =
+      job.experimental?.prebuildCommand ?? `prebuild --non-interactive --platform ${job.platform}`;
+    if (!prebuildCommand.match(/--platform/)) {
```

Comment:
```suggestion
    if (!prebuildCommand.match(/(?:--platform|-p)/)) {
```

Reply:
good catch, but `-p` could match sth else e.g. in template name

Reply:
Good point. Should probably include a whitespace before all flags for consistency?

Reply:
Or a word boundary
```suggestion
    if (!prebuildCommand.match(/(?:--platform|-p)\b/) {
```